components/layout/display_list/builder.rs

```diff
@@ -2272,10 +2277,15 @@ impl FragmentDisplayListBuilding for Fragment {
             DisplayListSection::Content,
         );
 
-        state.add_display_item(DisplayItem::Line(Box::new(LineDisplayItem {
-            base: base,
-            color: color.to_layout(),
-            style: LineStyle::Solid,
+        // TODO(gw): Use a better estimate for wavy line thickness.
+        let wavy_line_thickness = (0.33 * base.bounds.size.height).ceil();
+        state.add_display_item(DisplayItem::Line(CommonDisplayItem::new(
+            base,
+            webrender_api::LineDisplayItem {
+                orientation: webrender_api::LineOrientation::Horizontal,
+                wavy_line_thickness,
+                color: color.to_layout(),
+                style: LineStyle::Solid,
         })));
```

Comment:
nit: Indentation here feels a bit weird, probably the curly brace should be on its own line before this one now, and same above.